bedrock/settings/static_media.py

```diff
@@ -1221,13 +1250,6 @@
         ),
         'output_filename': 'js/firefox-features-hub-bundle.js',
     },
-    'firefox-features-landing': {
```

Comment:
Isn't this bundle still needed by the old page?

Reply:
🤦‍♂️